docs/gen-ai/gen-ai-metrics.md

```diff
@@ -266,8 +274,9 @@ This metric SHOULD be specified with [ExplicitBucketBoundaries] of
 | [`gen_ai.system`](/docs/attributes-registry/gen-ai.md) | string | The Generative AI product as identified by the client or server instrumentation. [2] | `openai` | `Required` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
 | [`error.type`](/docs/attributes-registry/error.md) | string | Describes a class of error the operation ended with. [3] | `timeout`; `java.net.UnknownHostException`; `server_certificate_invalid`; `500` | `Conditionally Required` if the operation ended in an error | ![Stable](https://img.shields.io/badge/-stable-lightgreen) |
 | [`server.port`](/docs/attributes-registry/server.md) | int | Server port number. [4] | `80`; `8080`; `443` | `Conditionally Required` If `sever.address` is set. | ![Stable](https://img.shields.io/badge/-stable-lightgreen) |
+| [`gen_ai.response.finish_reason`](/docs/attributes-registry/gen-ai.md) | string | The reason(s) the model stopped generating tokens. [5] | `stop`; `stop,length` | `Recommended` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
```

Comment:
@achandrasekar any thoughts/objections? on adding finish_reason attribute to LLM server metrics?

Reply:
This looks good to me. Adding a finish_reason attribute makes sense. It would be useful in a lot of cases including benchmarking where we can see if we generated to the max_length or stopped early due to the EOS token.